single_kernel_mongo/managers/ldap.py

```diff
@@ -205,10 +205,14 @@ def get_ldap_connection_status(self) -> StatusBase:
         bind_dn = self.state.ldap.bind_user
         bind_password = self.state.ldap.bind_password
         base_dn = self.state.ldap.base_dn
+        cert_chain = self.state.ldap.chain
 
         if not base_dn:
             return BlockedStatus("Missing base DN.")
```

Comment:
```suggestion
            return BlockedStatus("Missing base DN for ldap.")
```
What does the user need to do in this case?

Perhaps the needed user action can be added to the status

Also can you log something here

Reply:
I'll add some logs with some informations.
All thoses cases should never happen because we already checked that in the caller but I'll log some informations.